src/observers/link_click_observer.ts

```diff
@@ -71,10 +71,6 @@ export class LinkClickObserver {
   }
 }
 
-function doesNotTargetIFrame(anchor: HTMLAnchorElement): boolean {
-  for (const element of document.getElementsByName(anchor.target)) {
-    if (element instanceof HTMLIFrameElement) return false
-  }
-
-  return true
+function doesNotHaveSpecificTarget(anchor: HTMLAnchorElement): boolean {
+  return anchor.target != "_self"
```

Comment:
If we're going to have a special case handler for `_self`, could we also add test coverage to guarantee that behavior and guard against regressions?

Reply:
Done! BTW, I had to remove the check on `target` from `LinkClickObserver#findLinkFromClickTarget` to avoid a full-page load in the case of `[target=_self]`.